src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,62 @@
 package mate.academy;
 
+import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
-    public static void main(String[] args) {
-        MovieService movieService = null;
+    private static final Injector injector = Injector.getInstance("mate.academy");
```

Comment:
```suggestion
    private static final Injector INJECTOR = Injector.getInstance("mate.academy");
```
because it's a constant